src/main.ts

```diff
@@ -182,6 +184,72 @@ export function generateFile(ctx: Context, fileDesc: FileDescriptorProto): [stri
     chunks.push(generateDataLoadersType());
   }
 
+  if (options.outputSchema) {
+    chunks.push(code`
+      export interface ProtoMetadata {
+        fileDescriptor: ${imp('IFileDescriptorProto@protobufjs/ext/descriptor')};
+        references: { [key: string]: any };
+        dependencies?: ProtoMetadata[];
+      }
+    `);
+
+    const outputFileDesc: IFileDescriptorProto = {
+      ...fileDesc,
+      sourceCodeInfo: null,
+    };
+
+    if (fileDesc.sourceCodeInfo) {
+      outputFileDesc.sourceCodeInfo = {
+        location: fileDesc.sourceCodeInfo.location.filter((loc) => loc['leadingComments'] || loc['trailingComments']),
```

Comment:
Just curious, but what is the intent of this code? My guess is only outputing comments and not the `path` / `span` / etc. aspects of the source code location?

Reply:
Ah, I get it now, n/m.